src/parameters/all_parameters.cpp

```diff
@@ -428,6 +437,11 @@ void AllParameters::parse_parameters (dealii::ParameterHandler &prm)
     if (flux_reconstruction_aux_string == "kPlus")       { flux_reconstruction_aux_type = kPlus; }
     if (flux_reconstruction_aux_string == "k10Thousand") { flux_reconstruction_aux_type = k10Thousand; }
 
+    const std::string non_physical_behavior_string = prm.get("non_physical_behavior");
+    if (non_physical_behavior_string == "return_big_number")       { non_physical_behavior_type = return_big_number;}
```

Comment:
Consider replacing these assignments with 
`non_physical_behavior_type = NonPhysicalBehaviorEnum::return_big_number`... etc.

Reply:
For sure. I hadn't thought of doing that, but it is a little more clear. Fixed.